lib/config.js

```diff
@@ -0,0 +1,65 @@
+const path = require('path')
+
+const babel = {
+  presets: [
+    'babel-preset-env',
+    'babel-preset-stage-0',
+    'babel-preset-react',
+  ].map(require.resolve),
+  plugins: [
+    'babel-plugin-macros',
+    'babel-plugin-transform-runtime'
+  ].map(require.resolve)
+}
+
+const rules = [
+  {
+    test: /\.js$/,
+    exclude: /node_modules/,
+    loader: require.resolve('babel-loader'),
+    options: babel
+  },
+  {
+    test: /\.js$/,
+    exclude: path.resolve(__dirname, '../node_modules'),
+    include: path.resolve(__dirname),
+    loader: require.resolve('babel-loader'),
+    options: babel
+  },
+  {
+    test: /\.jsx$/,
+    loader: require.resolve('@compositor/jsx-loader'),
+    options: {}
+  },
+  {
+    test: /\.mdx$/,
```

Comment:
Yeah, I think these should be configurable via webpack.config.js – not sure the best approach to handling smart defaults and being able to disable them. I *think* that webpack-merge might combine rules intelligently but maybe not. Personally, I prefer *not* to use `.md` for `.mdx` since MDX isn't valid markdown and there might be other applications that try to open an MDX formatted `.md` file and break